src/VisualStudio/Core/Def/Implementation/ExtractInterface/ExtractInterfaceDialogViewModel.cs

```diff
@@ -186,6 +186,11 @@ public ImageSource Glyph
             {
                 get { return MemberSymbol.GetGlyph().GetImageSource(_glyphService); }
             }
+
+            public string MemberAutomationText
+            {
+                get { return "Member " + MemberName; }
```

Comment:
does this need to be localized? I don't know, so more of a clarification for myself than a suggestion.